acra/configuring-maintaining/logging/_index.md

```diff
@@ -1,4 +1,28 @@
 ---
-title: logging
+
+weight: 1
+title: Logging
 bookCollapseSection: true
 ---
+
+# Logging in Acra
+
+Starting with version [`0.77.0`](https://github.com/cossacklabs/acra/releases/tag/0.77.0), Acra supports three different logging formats: plaintext, [CEF](https://kc.mcafee.com/resources/sites/MCAFEE/content/live/CORP_KNOWLEDGEBASE/78000/KB78712/en_US/CEF_White_Paper_20100722.pdf), JSON.
+
+Acra logs are compatible with various external log analysing tools (like [ELK stack](https://www.elastic.co/elk-stack)) and SIEM systems.
+
+Logging mode and verbosity level can be configured for AcraServer, AcraTranslator and AcraWebConfig in the corresponding `yaml` files or passed as CLI parameter.
+
+Possible meanings for `logging_format` (case independent): 
+- plaintext
+- cef
+- json
+
+Verbosity level is enabled by changing `v` to `true`
+
+To see an example of logging, check out the default configuration of AcraServer in [`acra-server.yaml`](https://github.com/cossacklabs/acra-Q12021/blob/master/configs/acra-server.yaml#L99).
+
+{{< hint info >}}
```

Comment:
I'm not sure if this hint info is rendered as expected. At least browsing this README in displayed form

Reply:
This is a shortcut from Hugo that renders as highlighted line:

<img width="801" alt="Screenshot 2021-07-05 at 14 33 41" src="https://user-images.githubusercontent.com/2877920/124465388-072ea200-dd9e-11eb-8cc6-2749803a784d.png">